server/src/db/index.ts

```diff
@@ -212,3 +212,27 @@ export async function updateQrClaim(qr_hash: string, beneficiary:string, tx: Con
   });
   return res.rowCount === 1;
 }
+
+export async function createTask(task_name: string, task_data: string, api_key: string): Promise<Task|null> {
+
+  const now =  new Date();
+  const res = await db.result(
+    'SELECT * FROM task_creators WHERE task_name=${task_name} AND api_key=${api_key} AND valid_from < ${now} AND valid_to > ${now}',
+    {task_name, api_key, now}
```

Comment:
Check this: https://github.com/Xivis/poap/commit/bb17ffa12c43b9676134f704e41b84686ca335cf